todo-service/src/main/java/com/brtvsk/todoservice/migration/MongoChangelog.java

```diff
@@ -0,0 +1,39 @@
+package com.brtvsk.todoservice.migration;
+
+import com.brtvsk.todoservice.model.entity.Todo;
+import com.brtvsk.todoservice.repository.TodoRepository;
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@ChangeLog
+public class MongoChangelog {
+
+    @ChangeSet(order = "001", id = "testChange", author = "brtvsk")
+    public void testChangeLog(TodoRepository todoRepository) {
```

Comment:
is it possible to do it optional or configure it by feature flag

Reply:
Added migration profile